README.md

```diff
@@ -1,2 +1,25 @@
-# hw01
-Repository for homework 01
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# HOMEWORK 1
+
+__*Hello*__
+
+My name is Elaine Chen. I'm a third-year undergraduate student at the University of Chicago studying Political Science and Economics.
+
+![Portrait](https://raw.githubusercontent.com/elaineywchen/hw01/master/portrait%20for%20r.jpg){width=250px}
```

Comment:
I also encountered the problem that I did not know how to change the image size effectively. It's a pity that `{width=250px}` does not work as expected here.

Reply:
@elaineywchen @jiaqianwang18 Apparently, size changing is not working in markdown language. However, you can try - 

`<img src="https://raw.githubusercontent.com/elaineywchen/hw01/master/portrait%20for%20r.jpg" height = "100" width="200">`

This is html but it renders well in GitHub. You can use percentage symbol too. 

For example - 
`<img src="https://raw.githubusercontent.com/elaineywchen/hw01/master/portrait%20for%20r.jpg" height = "85%" width="90%">`